docs/contribute/saros-server.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Saros Server Development
+---
+
+## What is the Saros Server
+
+Please refer to the [User Documentation](../documentation/saros-server.md) for an explaination and the use-case of the saros server.
+
+Technically it is a headless session-host.
+
+## Code Structure
+
+The Saros Server codebase is comparably slim, as it only requires a [small subset of components](https://github.com/saros-project/saros/blob/master/server/src/saros/server/ServerContextFactory.java#L63) compared to IDE-Plugins. Therefor it may also help to understand Saros overall architecture and as a reference for starting new Saros-Implementations.
```

Comment:
Referencing a line has a high probability that the you reference the wrong code in the future.
```suggestion
The Saros Server codebase is comparably slim, as it only requires a [small subset of components](https://github.com/saros-project/saros/blob/master/server/src/saros/server/ServerContextFactory.java) compared to IDE-Plugins. Therefor it may also help to understand Saros overall architecture and as a reference for starting new Saros-Implementations.
```